src/shared/domain/repositories/activity_repository_interface.py

```diff
@@ -16,7 +16,7 @@ class IActivityRepository(ABC):
     @abstractmethod
     def get_enrollment(self, user_id: str, code: str) -> Enrollment:
         """
-        If the user is enrolled in the activity, returns the enrollment.
+        If the user_id is enrolled in the activity, returns the enrollment.
```

Comment:
If the user_id is enrolled in the activity?

Reply:
ok